tests/next_tests/unit_tests/embedded_tests/test_nd_array_field.py

```diff
@@ -683,14 +683,14 @@ def test_connectivity_field_inverse_image_2d_domain():
         codomain=V,
     )
 
-    # e2c_conn:
+    # e2v_conn:
```

Comment:
I see the problem but this is the wrong fix because e2v should always have only 2 neighbors. I think the whole test should be renamed to represent a c2v connectivity

Reply:
done